ruby/week_6_game/word_game.rb

```diff
@@ -0,0 +1,97 @@
+#Create a Word Game class
+#The class will have the following attributes:
+# -A word
+# -A count of the number of guesses made
+#  -A count of available guesses, which depends on the length of the word
+# -An array of the letters in the word
+# -A boolean denoting whether the game is over
+#The class will have the following instance methods:
+# -A method that checks if the guessed letter is in the word and keeps track of the amount of guesses (only adding if the letter hasn't been guessed already)
+# -A method that prints a string of blanks the length of the answer, and fills in letters as correct guesses are made
+#The driver code will do the following:
+# -Ask Player 1 for the word
+# -Print a statement that tells Player 2 how many guesses they have
+# -A loop that continually accepts guesses until the word is complete or they run out of guesses. Also prints the updated feedback after each guess, and how many guesses they've used
+# -A congratulatory message if they win, or a taunting message if they lose
+
+class WordGame
+  attr_reader :available_guesses, :guesses_made,  :game_over
+
+  def initialize(word)
+    @word = word.downcase
```

Comment:
i feel like this is a little over kill on the instance variables.

we should be just saving the state of the values we want.  line 21 seems fine.

if you want to iterate through the word characters i dont think you need an instance variable for that.

just call `@word.spit('').each etc...`  we are trying to manage the state and check values not make a tone of stuff to access